imageroot/systemd/user/timescaledb.service

```diff
@@ -0,0 +1,30 @@
+[Unit]
+Description=TimescaleDB for Samba Audit logs
+
+[Service]
+Environment=PODMAN_SYSTEMD_UNIT=%n
+EnvironmentFile=%S/state/environment
+WorkingDirectory=%S/state
+Restart=always
+TimeoutStopSec=70
+ExecStartPre=/bin/rm -f %t/%N.pid %t/%N.cid
+ExecStartPre=runagent write-timescaledb-config
+ExecStart=/usr/bin/podman run \
+    --conmon-pidfile %t/%N.pid \
+    --cidfile %t/%N.cid \
+    --cgroups=no-conmon \
+    --detach \
+    --network=host \
+    --replace --name %N \
+    --env=POSTGRES_DB=samba_audit \
+    --env=PGPORT=15432 \
```

Comment:
Not a fun an hardcoded port, but it should have no issue since we can have only one file server per node.